website/docs/admin_server_market.md

```diff
@@ -0,0 +1,44 @@
+---
+id: admin_server_market
+title: AYON Market
+sidebar_label: AYON Market
+description: Downloading addons and releases from the market.
+---
+
+:::info
+To access the market, you need to use AYON Cloud or have a local instance connected to Ynput Cloud.
+:::
+
+The AYON Market provides the latest official addons and releases, making it easy to enhance your server and pipeline. Access the market through the app menu or by pressing the shortcut `m+m` (double-tap the **m** key).
+
+![AYON Market in the app menu](./assets/server/admin/market/market-menu.png)
+
+## Release Bundles
+
+Release bundles include addons, dependency packages, and launchers tested by Ynput for compatibility. We recommend starting with a release bundle rather than individual addons.
+
+Bundles are tailored for different production tracking setups. You can deselect unnecessary pre-bundled addons before installation.
+
+![AYON Market release bundles](./assets/server/admin/market/market-release-bundles.png)
+
+After installing a release bundle, a new bundle is automatically created with your selected addons, launcher platforms, and pre-built, compatible dependency packages. We recommend testing the new bundle in a staging environment before using it in production. You can configure this in the Bundle Settings within Studio Settings.
```

Comment:
What about using colors for staging and production. 
Personally, It helps me connect this piece of information to bundles and launcher.
```suggestion
After installing a release bundle, a new bundle is automatically created with your selected addons, launcher platforms, and pre-built, compatible dependency packages. We recommend testing the new bundle in a <span style={{color:'#1c2026', backgroundColor:'#ff6446', borderRadius: '7px', padding: '3px'}}>staging</span> environment before using it in <span style={{color:'#1c2026',backgroundColor:'#00d7a1', borderRadius: '7px', padding: '3px'}}>production</span>. You can configure this in the Bundle Settings within Studio Settings.
```

![image](https://github.com/user-attachments/assets/ba9aa391-2ef7-4169-9937-f5c7979d789d)